src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration.component.ts

```diff
@@ -1,17 +1,20 @@
-import { Component, OnInit, TemplateRef, ViewChild } from '@angular/core';
+import { Component, TemplateRef, ViewChild } from '@angular/core';
 
 import { ConfigurationService } from '../../../shared/api/configuration.service';
+import { CdTableColumn } from '../../../shared/models/cd-table-column';
+import { CdTableSelection } from '../../../shared/models/cd-table-selection';
 
 @Component({
   selector: 'cd-configuration',
   templateUrl: './configuration.component.html',
   styleUrls: ['./configuration.component.scss']
 })
-export class ConfigurationComponent implements OnInit {
+export class ConfigurationComponent {
   @ViewChild('arrayTmpl') arrayTmpl: TemplateRef<any>;
```

Comment:
This should be removed, since it doesn't exist on the html and its value is always undefined.
Its usage should also be removed from the columns array.

If you want I can make this change in another PR, since you were not the one that added this code.

Reply:
I can remove it for this component or is there more that needs to be done?

Reply:
I could not find any other reference to this, so removing here should be enough.